app/boards/shields/stream_cheap/stream_cheap.zmk.yml

```diff
@@ -0,0 +1,8 @@
+file_format: "1"
+id: stream_cheap
+name: Stream Cheap Macropad
+type: shield
+url: https://www.thingiverse.com/thing:5231094
```

Comment:
This URL doesn't go anywhere as far as I can tell. Where can one purchase this shield?

Reply:
Hmm looks like an issue with thingiverse. Let me recreate it and see if i can access it externally without my account and update the link. In that thingiverse i included the files needed to upload it to JLCPCB or any other pcb manufacturer for pretty cheap since its a simple design.

Reply:
Resolved. Let me know if you can navigate to the url now. I was able to access it on my mobile service without logging in.